test/functionalTest/cases/1259_new_json_for_v2/put_attribute_value.test

```diff
@@ -307,7 +307,7 @@ Date: REGEX(.*)
 
 {
     "description": "Errors found in incoming JSON buffer",
-    "error": "BadRequest"
```

Comment:
This is step 2, which execution is:

```
echo '02. PUT /v2/entities/E1/attrs/A1/value with erroneous JSON payload'
echo '=================================================================='
payload='{ "A1" }'
orionCurl --url /v2/entities/E1/attrs/A1/value --payload "$payload" -X PUT --json
echo
echo
```

It is not exactly the same one that @raconte uses at https://github.com/telefonicaid/fiware-orion/issues/1428#issuecomment-173985143 but it should work...